From 1bfe3f4436d72cd3ef8bba20ca4b655112015c7b Mon Sep 17 00:00:00 2001
From: David Joy <davidjoy@users.noreply.github.com>
Date: Tue, 1 Dec 2020 10:04:59 -0500
Subject: [PATCH 0310/1021] Use prereqSectionName for the name of the prereq
 section. (#284)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Surprise, surprise.

Not sure how this wasn’t correct.  The bug results in the content lock UI showing the name of the current section rather than the prerequisite when describing… the prerequisite.
---
 src/courseware/course/sequence/SequenceContent.jsx      | 2 +-
 src/courseware/course/sequence/SequenceContent.test.jsx | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/courseware/course/sequence/SequenceContent.jsx b/src/courseware/course/sequence/SequenceContent.jsx
index c646713..51b796b 100644
--- a/src/courseware/course/sequence/SequenceContent.jsx
+++ b/src/courseware/course/sequence/SequenceContent.jsx
@@ -31,7 +31,7 @@ function SequenceContent({
         <ContentLock
           courseId={courseId}
           sequenceTitle={sequence.title}
-          prereqSectionName={sequence.gatedContent.gatedSectionName}
+          prereqSectionName={sequence.gatedContent.prereqSectionName}
           prereqId={sequence.gatedContent.prereqId}
         />
       </Suspense>
diff --git a/src/courseware/course/sequence/SequenceContent.test.jsx b/src/courseware/course/sequence/SequenceContent.test.jsx
index 23c60a2..bd2d9bb 100644
--- a/src/courseware/course/sequence/SequenceContent.test.jsx
+++ b/src/courseware/course/sequence/SequenceContent.test.jsx
@@ -32,7 +32,7 @@ describe('Sequence Content', () => {
     expect(screen.queryByText('Loading locked content messaging...')).not.toBeInTheDocument();
     expect(container.querySelector('svg')).toHaveClass('fa-lock');
     expect(screen.getByText(
-      `You must complete the prerequisite: '${gatedContent.gatedSectionName}' to access this content.`,
+      `You must complete the prerequisite: '${gatedContent.prereqSectionName}' to access this content.`,
     )).toBeInTheDocument();
     expect(screen.getByRole('button', { name: 'Go To Prerequisite Section' })).toBeInTheDocument();
   });
-- 
2.34.1

